### PR TITLE
[New Package] libfmt

### PR DESCRIPTION
**Pull request by saltedcoffii**

### CODE REVIEW

File: packages/libfmt.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+class Libfmt < Package
+  description 'A modern formatting library'
+  homepage 'https://fmt.dev'
+  version '7.1.3'
+  compatibility 'all'
+  source_url 'https://github.com/fmtlib/fmt/releases/download/7.1.3/fmt-7.1.3.zip'
+  source_sha256 '5d98c504d0205f912e22449ecdea776b78ce0bb096927334f80781e720084c9f'
+
+  def self.build
+    system "cmake #{CREW_CMAKE_OPTIONS} ."
```
Comment (uberhacker) [inline packages/libfmt.rb:12]: Typically, when building using cmake, a `build` directory is needed.  Did you follow the build instructions?

Comment (saltedcoffii) [reply packages/libfmt.rb:12]: cmake didn't respond with any errors or warnings when I built this

Comment (uberhacker) [reply packages/libfmt.rb:12]: That still doesn't answer my question. :)

Comment (saltedcoffii) [reply packages/libfmt.rb:12]: Yes, and they're (here)[https://fmt.dev/latest/usage.html#building-the-library]. Although the instructions say to make a `build` directory, they also say right above "The typical workflow starts with:", implying that it isn't necessary. You're welcome to add a build dir to this package script should you like tho :)

Comment (uberhacker) [reply packages/libfmt.rb:12]: The idea of the build directory is to isolate the build from the source.  That way the package doesn't end up containing extra files that are unnecessary.  I think this is a very important concept to understand.

Comment (saltedcoffii) [reply packages/libfmt.rb:12]: I didn't know that, thank you! A fix is on its way.

**Comment by satmandu** [comment]
Why not use Ninja with Cmake?
See https://github.com/satmandu/chromebrew/blob/master/packages/webkit2gtk.rb for an example. 

It makes for a simpler package, I think.

**Comment by saltedcoffii** [comment]
Apparently ninja has a negligible speed increase (between one and five seconds)
https://hamelot.io/programming/make-vs-ninja-performance-comparison/

Otherwise I don't know how using cmake's ninja generator makes the package simpler; I'm curious to know.

**Pull request merged**